packages/mobile/src/screens/upload-screen/utils/processTrackFile.ts

```diff
@@ -28,14 +29,25 @@ export const processTrackFile = (
     throw new Error('File must be an audio file')
   }
 
+  await TrackPlayer.add(
+    {
+      id: 'uploadTrack',
+      url: fileCopyUri ?? uri
+    },
+    0
+  )
+
```

Comment:
do we need to add the splits to `usdc_purchase` here, or are we going through the same upload sagas from web?

Reply:
naw since usdc_purchase is a nullable object, going to wait till that's selected before adding it into the form values, make sense?

Reply:
Oh I misunderstood where we are in the process. What I'm asking about should happen later on. Just curious if we are currently adding splits for mobile upload, but this is the wrong spot for it either way

Reply:
ah, so i added {} for splits to just have typecheck pass, but ideally going to use the same saga logic once we fire off upload, right?

Reply:
Yep! That's fine. See `processTracksForUpload`. If we're going through the same saga from web common, it's already there.